management/server/updatechannel.go

```diff
@@ -28,8 +31,15 @@ func (p *PeersUpdateManager) SendUpdate(peer string, update *UpdateMessage) erro
 	p.channelsMux.Lock()
 	defer p.channelsMux.Unlock()
 	if channel, ok := p.peerChannels[peer]; ok {
-		channel <- update
-		return nil
+		select {
+		case channel <- update:
+			log.Infof("update was sent to channel for peer %s", peer)
+			return nil
+		default:
+			msg := fmt.Errorf("channel for peer %s is %d full", peer, len(channel))
+			log.Info(msg)
+			return msg
```

Comment:
I wouldn't return error either because it can break sending messages to other peers while just one has the issue.